### PR TITLE
Docs: use correct @since version in multiple docblocks.

### PR DESCRIPTION
**Pull request by kebbet**
https://core.trac.wordpress.org/ticket/56792

### CODE REVIEW

**Comment by audrasjb** [comment]
Thanks for spotting this! 
One related commit: https://core.trac.wordpress.org/changeset/55010

**Comment by kebbet** [comment]
Another commit -> https://core.trac.wordpress.org/changeset/55345

**Comment by kebbet** [comment]
A third: https://core.trac.wordpress.org/changeset/54999

**Comment by kebbet** [comment]
And https://core.trac.wordpress.org/changeset/54973
Origin ticket was reopened for 6.1.2 https://core.trac.wordpress.org/ticket/57169#comment:5

**Comment by mukeshpanchal27** [comment]
Thanks @kebbet for PR. 

One move related commit: https://core.trac.wordpress.org/changeset/54894

**Review by mukeshpanchal27 — COMMENTED** [review]
Thanks @kebbet for PR.

One move related commit: https://core.trac.wordpress.org/changeset/54894

**Review by audrasjb — APPROVED** [review]
Ok let's go with this PR, thanks @kebbet!

File: tests/phpunit/tests/query/cacheResults.php [hunk 1]
```diff
@@ -1282,7 +1282,7 @@ public function data_author_cache_warmed_by_the_loop() {
 	/**
 	 * Ensure lazy loading term meta queries all term meta in a single query.
 	 *
-	 * @since 6.1.2
+	 * @since 6.2.0
```
Comment (mukeshpanchal27) [inline tests/phpunit/tests/query/cacheResults.php:1285]:
```suggestion
	 * @since 6.2.0
	 *
```

Add space between two tags.

**Comment by audrasjb** [comment]
Committed in https://core.trac.wordpress.org/changeset/55349

**Pull request closed without merging**